test-db.py

```diff
@@ -0,0 +1,36 @@
+from model import Model
+
+def create():
+    UserInfo = {'nutrition_standard': "fat", 'height': 170, 'weight': 37,
```

Comment:
change the name into `user_info`. Same for the two similar variables below.

Reply:
done